test/Tests/src/Data/Polyglot_Spec.enso

```diff
@@ -19,9 +21,26 @@ spec = Test.group "Polyglot" <|
         Polyglot.new String ["42"] . should_equal "42"
         Polyglot.new String ["42"].to_array . should_equal "42"
 
+    Test.specify "use Integer created by constructor" <|
+        Polyglot.new Integer [42] . should_equal 42
+
+    Test.specify "use Integer read from Polyglot object" <|
+        (Polyglot.get_member js_meaning "meaning") . should_equal 42
+
```

Comment:
```suggestion
        js_meaning.meaning . should_equal 42

```

Reply:
The goal of the code is to test `Polyglot.get_member` and make sure it properly coerces its value to `long`. Testing field read as well: 158309dc1